components/molecules/MostActiveContributorsCard/most-active-contributors-card.tsx

```diff
@@ -208,19 +220,36 @@ function RowTooltip({
   );
 }
 
-function GraphRow({ user, maxContributions }: { user: ContributorStat; maxContributions: number }) {
+function getWidthPercentage(stat: number, total_contributions: number) {
+  if (!stat || !total_contributions) {
+    return 0;
+  }
+
+  return (stat / total_contributions) * 100;
+}
+
+function GraphRow({
+  user,
+  maxContributions,
+  dataLabels,
+}: {
+  user: ContributorStat;
+  maxContributions: number;
+  dataLabels: Record<StatKeys, DataLabel>;
+}) {
+  const keys = Object.keys(dataLabels) as StatKeys[];
   const [isHovered, setIsHovered] = useState(false);
   const [springs] = useSprings(
-    dataTypes.length,
+    keys.length,
```

Comment:
The changes in the graph row are related to the new properties on the `ContrutorStat` as well as adjusting which contribution types to surface in the graph as we currently don't have all of it backfilled yet.